Sources/SwiftParser/Modifiers.swift

```diff
@@ -167,4 +148,66 @@ extension Parser {
 
     return RawDeclModifierSyntax(name: keyword, detail: detail, arena: self.arena)
   }
+
+  mutating func parseAccessLevelModifier() -> RawDeclModifierSyntax {
+    assert(
+      [
+        RawTokenKind.privateKeyword,
+        .fileprivateKeyword,
+        .internalKeyword,
+        .publicKeyword
+      ].contains(currentToken.tokenKind)
+    )
+    let name = consumeAnyToken()
+    let details: RawDeclModifierDetailSyntax?
+    if let lparen = consume(if: .leftParen) {
+      var beforeDetail: [RawSyntax] = []
+      var detail: RawTokenSyntax? = nil
+      var afterDetail: [RawSyntax] = []
+
+      let rightParenPrecedence = TokenPrecedence(.rightParen)
+
+      // Find `set` keyword during eat all tokens as junk
+      // until reach to right-paren or other strong token.
+      while !at(.eof),
+            TokenPrecedence(currentToken.tokenKind) < rightParenPrecedence
+      {
+        if currentToken.isContextualKeyword("set") {
+          detail = consume(remapping: .contextualKeyword)
+          break
+        }
+        beforeDetail.append(RawSyntax(consumeAnyToken()))
+      }
+
+      // Eat rest of junk and right-paren
+      let (junk, rightParen) = expect(.rightParen)
+      if let junk = junk {
+        afterDetail.append(contentsOf: junk.elements)
+      }
+      
+      details = RawDeclModifierDetailSyntax(
+        leftParen: lparen,
+        beforeDetail.isEmpty ? nil :
+          RawUnexpectedNodesSyntax(
+            elements: beforeDetail, arena: arena
+          ),
+        detail: detail ?? RawTokenSyntax(
+          missing: .contextualKeyword,
+          text: arena.intern("set"),
```

Comment:
`arena.intern("set")` is not necessary here because `"set"` is a `StaticString` that doesn’t get deallocated because it lives in the binary itself (instead of stack or heap memory). Thus, it doesn’t need to be managed by a `SyntaxArena`.

Reply:
When I simply wrote `text: "set"` here,
I got following error:

```
SwiftSyntax/RawSyntax.swift:427: Assertion failed: token text must be managed by the arena, or known default text for the token
2022-09-08 12:00:12.816402+0900 swift-parser-test[59612:27324768] SwiftSyntax/RawSyntax.swift:427: Assertion failed: token text must be managed by the arena, or known default text for the token
```

It come from this assertion.
https://github.com/apple/swift-syntax/blob/1754a24d6f5aebe39cd066b5d17438621ea02009/Sources/SwiftSyntax/Raw/RawSyntax.swift#L427-L428

How do I write?

Reply:
Ah, I see. Makes sense. I’ll probably remove that `assert` in another PR but you can just call `arena.intern` for now.